compiler/src/main/java/com/squareup/anvil/compiler/InterfaceMerger.kt

```diff
@@ -64,11 +64,13 @@ internal class InterfaceMerger(
           )
         }
       }
+      // Convert the sequence to a list to avoid iterating it twice. We use the result twice
+      // for replaced classes and the final result.
+      .toList()
 
     val replacedClasses = classes
       .flatMap { (classDescriptor, contributeAnnotation) ->
         contributeAnnotation.replaces(classDescriptor.module)
-          .asSequence()
```

Comment:
Most of the time `replaces(..)` returns an empty list. Using the `replaces` annotation attribute should be rare. It's unnecessary to convert an empty list to a sequence.